modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -430,9 +435,21 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
                  " JOIN test_names t ON (t.Test_name = f.Test_name)".
                  " JOIN psc ON (s.CenterID = psc.CenterID)";
 
-        $where   = " WHERE s.Active = 'Y' AND c.Active = 'Y'".
+        $where = " WHERE s.Active = 'Y' AND c.Active = 'Y'".
                    " AND coalesce(f.data_entry,'In Progress') = 'In Progress'".
                    " AND psc.Centerid!= '1' AND c.Entity_type != 'Scanner'";
+
+        if (count($ddeInstruments) > 0) {
+            $ddeInstruments = implode(",", $ddeInstruments);
+            $where         .= "
+                AND (f.test_name IN ($ddeInstruments) OR
+                f.commentid NOT LIKE 'DDE_%')
+            ";
+        } else {
+            $where .= "
+                AND (f.commentid NOT LIKE 'DDE_%')
+            ";
```

Comment:
I think this logic looks right to me, but I'm not really familiar with how the module uses the query either.

Can someone who knows the module test this before we merge it?